lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -4,10 +4,6 @@ namespace :update do
       Dir.chdir engine.path do
         next unless File.file? 'package.json'
 
-        if ENV['TEST_SUITE'] == 'spec'
```

Comment:
You probably don't need to remove this anymore, there are no more yarn-installed assets going through the asset pipeline (as of last week, only patternfly remains, and that comes from gems).

Reply:
I'll try it out

Reply:
I'm getting this without yarn:

```
  2) ContainerImageRegistryController download pdf file title is set correctly
     Failure/Error: MiqServer.my_server.name
     
     ActionView::Template::Error:
       undefined method `name' for nil:NilClass
     Shared Example Group: "#download_summary_pdf" called from ./spec/controllers/container_image_registry_controller_spec.rb:58
     # ./app/helpers/application_helper.rb:1238:in `appliance_name'
     # ./app/views/layouts/_user_options.html.haml:10:in `__home_travis_build_jrafanie_manageiq_ui_classic_app_views_layouts__user_options_html_haml___1586482537770873932_402158000'
     # ./app/views/layouts/_header.html.haml:32:in `__home_travis_build_jrafanie_manageiq_ui_classic_app_views_layouts__header_html_haml___583429950045279110_426101180'
     # ./app/views/layouts/application.html.haml:32:in `__home_travis_build_jrafanie_manageiq_ui_classic_app_views_layouts_application_html_haml___2959479064510394380_257727600'
     # ./app/controllers/application_controller.rb:280:in `block (2 levels) in render_exception'
     # ./app/controllers/application_controller.rb:268:in `render_exception'
     # ./app/controllers/application_controller.rb:263:in `error_handler'
     # ./spec/shared/controllers/shared_example_for_download_summary_pdf.rb:12:in `block (3 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # Sprockets::FileNotFound:
     #   couldn't find file '@manageiq/ui-components/dist/css/ui-components' with type 'text/css'
     #   Checked in these paths: 
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/manageiq-v2v-f2e1b8c433a4/app/assets/config
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/manageiq-v2v-f2e1b8c433a4/app/assets/images
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/manageiq-v2v-f2e1b8c433a4/app/assets/stylesheets
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/jquery-rjs-ded3d28d0049/vendor/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/manageiq-decorators-19780e2776c8/app/assets/images
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/gettext_i18n_rails_js-1.3.0/lib/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/gettext_i18n_rails_js-1.3.0/vendor/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/app/assets/images
     #     /home/travis/build/jrafanie/manageiq-ui-classic/app/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/app/assets/stylesheets
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/novnc-rails-0.2/vendor/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/coffee-rails-4.2.2/lib/assets/javascripts
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/actioncable-5.1.7/lib/assets/compiled
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/gems/actionview-5.1.7/lib/assets/compiled
     #     /home/travis/build/jrafanie/manageiq-ui-classic/vendor/bundle/bundler/gems/manageiq-decorators-19780e2776c8/app/assets/images
```

https://travis-ci.org/jrafanie/manageiq-ui-classic/jobs/544952854

Reply:
the failures are all in the download_pdfs tests @himdel 

```
rspec ./spec/controllers/provider_foreman_controller_spec.rb:709 # ProviderForemanController download pdf file request returns 200
rspec ./spec/controllers/container_image_registry_controller_spec.rb[1:5:2] # ContainerImageRegistryController download pdf file title is set correctly
rspec ./spec/controllers/container_image_registry_controller_spec.rb[1:5:1] # ContainerImageRegistryController download pdf file request returns 200
rspec ./spec/controllers/container_image_registry_controller_spec.rb[1:5:3] # ContainerImageRegistryController download pdf file should not raise error
rspec ./spec/controllers/cloud_subnet_controller_spec.rb[1:7:1] # CloudSubnetController download pdf file request returns 200
rspec ./spec/controllers/container_image_controller_spec.rb[1:8:1] # ContainerImageController download pdf file request returns 200
rspec ./spec/controllers/container_image_controller_spec.rb[1:8:2] # ContainerImageController download pdf file title is set correctly
rspec ./spec/controllers/container_image_controller_spec.rb[1:8:3] # ContainerImageController download pdf file should not raise error
rspec ./spec/controllers/configuration_job_controller_spec.rb[1:3:1] # ConfigurationJobController download pdf file request returns 200
rspec ./spec/controllers/vm_infra_controller_spec.rb[1:41:1] # VmInfraController download pdf file request returns 200
rspec ./spec/controllers/vm_infra_controller_spec.rb[1:42:1] # VmInfraController download pdf file request returns 200
rspec ./spec/controllers/container_node_controller_spec.rb[1:6:1] # ContainerNodeController download pdf file request returns 200
rspec ./spec/controllers/container_node_controller_spec.rb[1:6:2] # ContainerNodeController download pdf file title is set correctly
rspec ./spec/controllers/container_node_controller_spec.rb[1:6:3] # ContainerNodeController download pdf file should not raise error
rspec ./spec/controllers/container_group_controller_spec.rb[1:5:1] # ContainerGroupController download pdf file request returns 200
rspec ./spec/controllers/container_group_controller_spec.rb[1:5:2] # ContainerGroupController download pdf file title is set correctly
rspec ./spec/controllers/container_group_controller_spec.rb[1:5:3] # ContainerGroupController download pdf file should not raise error
```

Reply:
Aaah, that's because we still have...

```
app/assets/stylesheets/print.scss
4: *= require @manageiq/ui-components/dist/css/ui-components
```

I'm creating a print js pack to replace the print stylesheet in https://github.com/ManageIQ/manageiq-ui-classic/pull/5679 so this will go away.

Reply:
Ok, I'll put the commit back and reference that PR. Thanks!

Reply:
FYI, resolved by https://github.com/ManageIQ/manageiq-ui-classic/pull/5722